docker/scripts/dev_start.sh

```diff
@@ -439,14 +439,8 @@ function mount_map_volumes() {
 function mount_other_volumes() {
     info "Mount other volumes ..."
     local volume_conf=
-    if [ "${FAST_MODE}" = "no" ]; then
-        # YOLO3D
-        local yolo3d_volume="apollo_yolo3d_volume_${USER}"
-        local yolo3d_image="${DOCKER_REPO}:yolo3d_volume-${TARGET_ARCH}-latest"
-        docker_restart_volume "${yolo3d_volume}" "${yolo3d_image}"
-        volume_conf="${volume_conf} --volumes-from ${yolo3d_volume}"
-    fi
-
+    # if [[ "${FAST_MODE}" == "no" ]]; then
```

Comment:
why still leave these commented lines?

Reply:
Intended for placeholder for other non-fast-mode volumes. Although It seems that nearly every volume is in-fast-mode.

Reply:
> why still leave these commented lines?

Cleaned up.